docs/db/week-3/lesson.md

```diff
@@ -508,6 +509,51 @@ Note also that optimistic locking requires changes not only to the code of your
 
 ---
 
+## PD (CVs and transferable skills)
+
+**Session objective:** This session focuses on identifying transferable skills and designing effective CVs, Linkedin and GitHub profiles.
+
+[CVs and transferable skills slides](https://docs.google.com/presentation/d/1sXfvLTImKKJo7IdvKwL7--beKygBseMvD12Y_wDKHmY/edit#slide=id.ga9333c68a9_0_33)
+
+:::note Exercise (10 min)
+
+**Exercise objective:** To identify transferrable skills from different life experiences.
+List 3 transferable skills to be gained for each of the following jobs:
```

Comment:
```suggestion
List 3 transferable skills to be gained for each of your previous jobs or educational experiences:
```

Reply:
Should we use the trainees' experience, rather than generic examples of other jobs? Most of them have done some kind of job before. And if they haven't, they can still use educational experience or think about someone close to them?